data.sql

```diff
@@ -133,4 +133,4 @@ INSERT INTO
         weight_kg
     )
 VALUES
-    ('Ditto', '2022-05-14', 4, TRUE, 17);
\ No newline at end of file
+    ('Ditto', '2022-05-14', 4, TRUE, 17); 
```

Comment:
- [ ] [REQUIRED] Good job so far!  Please give the correct value to `weight_kg` 🙏🏼 
```SQL
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Ditto', '2022/05/14', '4', 'true', '22.0');
```